tests/components/pyload/test_button.py

```diff
@@ -26,7 +26,7 @@
 
 
 @pytest.fixture(autouse=True)
-async def button_only() -> AsyncGenerator[None, None]:
+async def button_only() -> AsyncGenerator[None]:
```

Comment:
Doesn't need to be async
```suggestion
def button_only() -> Generator[None]:
```